docs-snippets/message-persistence.ts

```diff
@@ -0,0 +1,51 @@
+import PubNub from '../lib/types';
+
+const pubnub = new PubNub({
+  publishKey: 'demo',
+  subscribeKey: 'demo',
+  userId: 'myUniqueUserId',
+});
+
+// snippet.fetchMessagesWithMetaActions
+try {
+  const response = await pubnub.fetchMessages({
+    channels: ['my_channel'],
+    stringifiedTimeToken: true,
+    includeMeta: true,
+    includeMessageActions: true,
+    includeCustomMessageType: true,
+  });
+  console.log(`fetch messages response: ${response}`);
+} catch (status) {
+  console.log(`fetch messages failed with error: ${status}`);
```

Comment:
Same issues in this file as in previous with how object will be printed in formatted string and `status` instead of `error`.

Reply:
[message-persistence.ts](https://github.com/pubnub/javascript/blob/docs-snippets/docs-snippets/basic-usage/message-persistence.ts) updated